pyodata/v2/service.py

```diff
@@ -235,11 +235,11 @@ def __repr__(self):
 class ODataHttpRequest:
     """Deferred HTTP Request"""
 
-    def __init__(self, url, connection, handler, headers=None):
+    def __init__(self, url, connection, handler, headers={}):
         self._connection = connection
         self._url = url
         self._handler = handler
-        self._headers = headers
+        self.headers = headers
```

Comment:
The changes happen in the following lines

Reply:
Do you really need to make the member public? If so, I would prefer a property.

Reply:
Gotcha, this way we can also have a set_headers method that can append/update headers.